src/custom-elements/src/components/label/label.tsx

```diff
@@ -38,7 +49,37 @@ export class Label {
     addMutationObserver.call(this);
   }
 
+  componentWillLoad() {
+    this.helpButtonId = `help-button-${uuid4()}`;
+    this.helpPopoverId = `help-popover-${uuid4()}`;
+  }
+
+  componentDidLoad(): void {
+    this.el
+      .querySelector(`#${this.helpButtonId}`)
+      .addEventListener('click', () => (document.querySelector(`#${this.helpPopoverId}`) as any).toggle());
```

Comment:
Could we use 

```suggestion
     <chi-button onChiClick="() => this.el.querySelector(`#${this.helpPopoverId}`) as any).toggle()"
```

in the render function? Traversing the DOM is usually slower